run.sh

```diff
@@ -1,16 +1,17 @@
-#!/bin/sh
+#!/bin/bash
 
-GPG=$HOME/gnupg-1.4.12/bin/gpg
-ADDR=addr/osx.txt
```

Comment:
This section is starting to become a bit of a back-and-forth between OS X and Linux-specific code. (I know, I know, I should have done this first, but I was pretty terrible at shell scripting 1.5 years ago.) Let's replace this with something like this:

```bash
# Use a different set of default addresses depending on OS X vs Linux
case "$(uname -s)" in
  Darwin)
    # Mac OS X (or soon-to-be MacOS)
    ADDR=docs/addr/osx.txt
    ;;
  Linux)
    ADDR=docs/addr/linux.txt
    ;;
  *)
    echo "OS is not Darwin or Linux-based. Defaulting to Linux addresses."
    ADDR=docs/addr/linux.txt
    ;;
esac
```

If you're interested in spending a bit more time with this, you can also have this take an optional argument with the path to the address file, e.g. `ADDR=$1`, but default to OS detection.